examples/parameters.rs

```diff
@@ -7,6 +7,32 @@
 //! Note that because `ignore_seek_speed` is `false` in these examples,
 //! it could take a few seconds before you can hear the changes in these sound effects.
 //! See https://www.fmod.com/docs/2.02/studio/parameters-reference.html for more details.
+//!
+//! # Global parameters
+//!
+//! The FMOD example project unfortunately does not have any global parameters, so we can't include
+//! an example of that here.
```

Comment:
```suggestion
//! The FMOD example project unfortunately does not have any global parameters, so we can't include
//! a runnable example of that here.
```

Just a quick suggestion to make the distinction between local parameters and global parameters in this file